src/main/scala/centaur/test/Test.scala

```diff
@@ -151,6 +151,23 @@ object Operations {
     }
   }
 
+  def validateDirectoryContentsCounts(workflowDefinition: Workflow, workflowId: String): Test[Unit] = new Test[Unit] {
+    override def run: Try[Unit] = workflowDefinition.directoryContentCounts match {
+      case None => Success(())
+      case Some(directoryContentCountCheck) =>
+        val counts = directoryContentCountCheck.expectedDrectoryContentsCounts map {
```

Comment:
There's no "I" in team!

... but there is one in `expectedDirectoryContentsCounts`